explorer/app/apis/catalog/ncpi-catalog/common/utils.ts

```diff
@@ -30,81 +32,59 @@ export function buildNCPICatalogPlatforms(ncpiCatalogs: unknown[]): unknown[] {
   return [...ncpiCatalogPlatformsByPlatform.values()];
 }
 
-/**
- * Returns NCPI catalog studies.
- * @param ncpiCatalogs - NCPI catalogs.
- * @returns NCPI catalog studies.
- */
-export function buildNCPICatalogStudies(ncpiCatalogs: unknown[]): unknown[] {
-  return (ncpiCatalogs as NCPICatalog[]).map((ncpiCatalog) => {
-    const study: NCPICatalogStudy = {
-      consentCodes: ncpiCatalog.consentCodes,
-      dataTypes: ncpiCatalog.dataTypes,
-      dbGapId: ncpiCatalog.dbGapId,
-      focusDisease: ncpiCatalog.focusDisease,
-      participantCount: ncpiCatalog.participantCount,
-      platforms: ncpiCatalog.platforms,
-      studyAccession: ncpiCatalog.studyAccession,
-      studyDesigns: ncpiCatalog.studyDesigns,
-      studyName: ncpiCatalog.study,
-    };
-    return study;
-  });
-}
-
 /**
  * Returns NCPI catalog platform.
  * @param platform - The platform for the given NCPI catalog platform.
- * @param ncpiCatalogStudy - NCPI catalog study.
+ * @param ncpiPlatformStudy - a platform study
  * @param ncpiCatalogPlatform - NCPI catalog platform.
  * @returns NCPI catalog platform.
  */
 function buildNCPICatalogPlatform(
```

Comment:
Review all property names within this function `buildNCPICatalogPlatform` now that some property names have been reverted.